### DISCUSSION TITLE
Is SSD necessary?

### DISCUSSION THREAD

**Discussion by Chaython** [Chia-Network/chia-blockchain#4237] [Q&A]
I saw in this calculator, if you select 2tb, it suggest the only hardware needed is a 2tb hdd and you'd make $200/month.
https://chiacalculator.com/
But most talk I see is about SSDs.
So what's the difference in performance between ssd and hdd in mining chia?
I have 6 old hard drives which = 13tb, how much payment is realistic?
Currently 1tb m.2 ssds are $130 locally, how much benefit would there be from $1000 in ssds?

**Comment by cccat6** [comment]
You need to read through the documents of Chia. Seems you completely have no idea what Chia is and how it works.
Please be clear about what is plotting and what is farming. Then you will get the answer. Chia is NOT donating your space and done. You need to plot and fill your drive. Then you can use the plot files to farm. Thus you can earn Chia.
SSD is for speeding up plotting, not for storage. We don't care HDD since there is not much to discuss, it just storage.

**Reply by Chaython** [reply]
So HDD is bad? But when you select 2tib in the calc it suggest hardware cost as hdd.
LMK ssds are still really cheap in Canada, but this mining probably kills them in a few months.

**Reply by cccat6** [reply]
We use SSD plotting, and use HDD to store the plot file. That's what we usually do.
Plotting will kill SSD, but farming does not. However, HDD is enough for farming, you do not need expensive SSD just for storing these plots.
HDD can plot for sure, but it's slow. SSD can finish multiple plot tasks in 8 hours. But HDD can only make a single task in more than 12 hours. That's the difference.
